src/xdocs/config_annotation.xml

```diff
@@ -142,13 +142,20 @@ public String getNameIfPresent() { ... }
 
       <subsection name="Examples" id="AnnotationLocation_Examples">
         <p>
-          Example to allow multiple annotations on the same line
+          Default configuration:
         </p>
         <source>
-@SuppressWarnings("deprecation") @Mock DataLoader loader; // no violations
+&lt;module name=&quot;AnnotationLocation&quot;/&gt;
+        </source>
+        <p>
+          Example for default configuration
+        </p>
+        <source>
+@Override public int hashCode() { ... } // no violations
+@SuppressWarnings("deprecation") public int foo() { ... } // violation
         </source>
         <p>
-          Use the following configuration:
+          Use the following configuration to allow multiple annotations on the same line:
```

Comment:
`allowSamelineSingleParameterlessAnnotation` is also different than the original value, so the text should be updated to include that.

Reply:
I have already added "allowSamelineSingleParameterlessAnnotation"